src/openssl_wrap.cpp

```diff
@@ -414,6 +414,8 @@ void _X509_REQ_sign_ctx(X509_REQ* req, EVP_MD_CTX* ctx)
 const EVP_MD* _getMDPtrFromDigestType(DigestTypes type)
 {
     switch (type) {
+        case DigestTypes::NONE:
```

Comment:
This function used to return only valid objects and otherwise throw. I think this is the more reasonable behavior than returning nullptrs if you want things to fail loudly.

Reply:
I this case nullptr does not indicate an error, instead it is expected by openssl to be passed if no hashing shall be used. (See e.g. https://www.openssl.org/docs/man1.1.1/man3/EVP_DigestSignInit.html)